plugins/uniques.js

```diff
@@ -13,14 +12,18 @@ window.plugin.uniques.SYNC_DELAY = 5000;
 
 // maps the JS property names to localStorage keys
 window.plugin.uniques.FIELDS = {
-	'uniques': 'plugin-uniques-data',
-	'updateQueue': 'plugin-uniques-data-queue',
-	'updatingQueue': 'plugin-uniques-data-updating-queue',
+  'uniques':       'plugin-uniques-data',
```

Comment:
Please do not change indents in old code that does not affect your PR

Reply:
I was previously asked to take care of 2-spaces indentation and not use tabs. So the code was re-formated in all sections/functions that received new or changed code.

Reply:
I suggest you to preserve existing whitespace style otherwise it gets messed and it's hard to review the actual changes.
If we need to change whitespace - we will do it at once in the whole file.

Reply:
As the nesting was changed the code unindented. Do the comparison with "ignore white space" and you get what you want.

Reply:
Well, I do not see any indent change in final diff, so nothing to discuss anymore